va-gov/includes/breadcrumbs.html

```diff
@@ -5,8 +5,15 @@
     <li>
       <a href="/" onClick="recordEvent({ event: 'nav-breadcrumb', 'nav-breadcrumb-section': 'home' });">Home</a>
     </li>
-    {% assign current_page = breadcrumb_path | last %}
-    {% for crumb in breadcrumb_path %}
+    {% assign crumbs = breadcrumb_path %}
+
+    {% if breadcrumbs_override %}
+    {% assign crumbs = breadcrumbs_override %}
+    {% endif %}
+
+    {% assign current_page = crumbs | last %}
+
+    {% for crumb in crumbs %}
```

Comment:
This was implemented because we need to borrow the styles from the detail-page template and a method to override the breadcrumbs within it, because the brand-consolidated GI Bill Benefits Check has breadcrumbs that aren't generated by its page path.